.gitignore

```diff
@@ -56,6 +58,7 @@ src/qt/bitcoin-qt.creator
 src/qt/bitcoin-qt.creator.user
 src/qt/bitcoin-qt.files
 src/qt/bitcoin-qt.includes
+contrib/bitcoin-qt.pro.user
```

Comment:
I guess thats the Qt Creator project file? Why would that be stored in contrib? (Asking because mine is in the root folder AFAIK).

Reply:
Good question. I will try to determine why. Thank.